layouts/shortcodes/swagger-ui.html 

```diff
@@ -0,0 +1,9 @@
+<div id="swagger-ui-wrapper" data-url="{{.Get 0}}"></div>
```

Comment:
We should also apply the concept everywhere, to be fully agnostic about our dependencies... shortcode and also the identifiers we use should be rename to `apiDocs` and `api-docs` where they apply.